### ISSUE TITLE
Does modifying the buildcraft silicon jars json files and adding new ones create new recipes?

### ISSUE THREAD

**Issue by CrankySupertoonArchive** [BuildCraft/BuildCraft#4258]
Does this work, and if this does work can it be a config file too so i dont have to redistrubte the jar?

**Comment by CrankySupertoon01** [comment]
Or does making a resource pack with the json files work? I honestly dont know

**Comment by CrankySupertoon01** [comment]
Also if you wouldnt mind me asking. Buildcraft is in beta or release but its flagged as alpha on curse so people dont break the mj api

**Comment by CrankySupertoon01** [comment]
Also it seems that the Buildcraft compat module still doesnt read assembly recipes

**Comment by LuigiHutch** [comment]
No, BC is still in alpha (as marked on curse) and will stay in alpha till all the features that are being reworked on are complete and the mod is feature complete compared to 1.7

The BC compat module for 1.12 does not have JEI integration to display the assembly table recipes and it will not be added till the assembly table api is rewritten in MJ update so it will be added sometime after that.

Please use the discord for random questions such as these next time: https://discord.gg/BuildCraft
